src/reset.css

```diff
@@ -0,0 +1,17 @@
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body, h1, h2, h3, h4, h5, h6, p, ol, ul, form {
```

Comment:
I have a good option for you , see this style 👍  : * { // padding: 0; margin: 0 }